ReactNativeTraining/src/app/screens/chats/Chats.js

```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from 'react';
```

Comment:
Components and Screens components folders must name with PascalCase
`screens/Chats/index.js`